middleware.js

```diff
@@ -265,18 +269,18 @@ module.exports = function(options) {
 
     // Compare mtimes
     fs.stat(sassPath, function(err, sassStats) {
-      if (err) {
-        error(err);
```

Comment:
If I understand the code right, `sassPath` is computed from the request – it's the path to the potentially existing `.scss` file which, if it's there and younger than the `.css` file, will trigger compilation. Notice in the original code the error is logged with `error(err)` but then we just let `next()` handle the request.

Inside `compile()`, which gets called after this code, we check that `sassPath` `fs.exists` - that seems redundant now; or it indicates I don't quite get what's going on.